debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerStreamingChangeEventSource.java

```diff
@@ -647,16 +647,16 @@ private Scn calculateEndScn(OracleConnection connection, Scn startScn, Scn prevE
         streamingMetrics.setCurrentScn(currentScn);
 
         // Add the current batch size to the starting system change number
```

Comment:
@vjuranek Does the comment still reflect the code?

Reply:
yes, it still applies, but let's remove it as it just describes the code instead of explaining the code, so it's completely useless